src/room.ts

```diff
@@ -209,23 +231,47 @@ export class Room extends EventEmitter implements Sayable {
     return rawMemberList.map(m => Contact.load(m.UserName))
   }
 
+  /**
+   * ISSUE #64 emoji need to be striped
+   * ISSUE #104 never use remark name because sys group message will never use that
+   * @rui: cannot use argument NickName because it mix real nick and remark 
+   */
   private parseNickMap(memberList: RoomRawMember[]): Map<string, string> {
     const nickMap: Map<string, string> = new Map<string, string>()
-
+    let contact: Contact
     if (memberList && memberList.map) {
       memberList.forEach(member => {
-        /**
-         * ISSUE #64 emoji need to be striped
-         * ISSUE #104 never use remark name because sys group message will never use that
```

Comment:
Please keep the original comments in the code, and add new comments if needed, for keep the history clear.(until someday we decide to clean them)